docs/guide/development/documentation.md

```diff
@@ -34,6 +34,16 @@ If your module wasn't already in the reference docs, update `docs/.vuepress/conf
 }
 ```
 
+### Archiving API docs
+
+Whenever a new release of Prefect is cut, we must archive the old API docs so they are available for users using the older versions.  To do so, follow these steps:
+- first, make sure you are on the release commit: `git checkout $VERSION_TAG`
+- next, generate the documentation: `cd docs/ && python generate_docs.py`
```

Comment:
not critical whatsoever, but we now also have `yarn docs:generate-api` for this step

Reply:
Good to know, somehow I missed that!  Since ultimately you have to work in the `docs/` directory anyway, I'm just going to leave it as is.